core/perf_test/BenchmarkMain.cpp

```diff
@@ -44,4 +44,18 @@
 
 #include <benchmark/benchmark.h>
 
-BENCHMARK_MAIN();
+#include <Benchmark_Context.hpp>
+#include <Kokkos_Core.hpp>
+
+int main(int argc, char** argv) {
+  Kokkos::initialize(argc, argv);
+  benchmark::Initialize(&argc, argv);
+  benchmark::SetDefaultTimeUnit(benchmark::kSecond);
+  Test::add_benchmark_context(true);
```

Comment:
```suggestion
  KokkosBenchmark::add_benchmark_context(true);
```
In my previous comment I implicitly meant to update this as well

Reply:
Ah, yes I forgot that there is also that namespace used. Thanks!